### PR TITLE
feat(datepicker): add presets story to datepicker component and fix calendar and chips css

### PR DESCRIPTION
**Pull request by veekays**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/patterns/datePicker/datePickerWithPresets.story.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+              setDate(DatePicker.utils.getTomorrowDate())
+              setSelectedChip('tomorrow')
+            }}
+          />
+          <Chip
+            label="3 Days later"
+            clearButton={false}
+            type="selection"
+            className="mb-5  d-block"
+            selected={selectedChip === '3DaysLlater'}
```
Comment (xlreon) [inline core/components/patterns/datePicker/datePickerWithPresets.story.tsx]: Small l to be removed

File: core/components/patterns/datePicker/variants/smallDatePickerWithPresets.story.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { DatePicker } from '@/index';
+
+const customCode = `() => {
+  const [date, setDate] = React.useState(new Date());
+  const [selectedChip, setSelectedChip] = React.useState('today');
+
+  const renderChildren = () => {
+    return (
+      <div className="pt-6 px-5" style={{ borderRight: 'var(--border)', borderColor: 'var(--secondary-light)' }}>
```
Comment (xlreon) [inline core/components/patterns/datePicker/variants/smallDatePickerWithPresets.story.tsx]: borderRight use design system class

File: core/components/organisms/datePicker/utilities.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+const setDate = (jumpDateIndex: number) => {
+  const d = new Date();
+  return d.setDate(d.getDate() + jumpDateIndex);
+};
+
+export const getTomorrowDate = () => {
```
Comment (xlreon) [inline core/components/organisms/datePicker/utilities.tsx]: Assertions to be modified

File: core/components/organisms/datePicker/DatePicker.tsx [hunk 4]
```diff
@@ -257,5 +265,12 @@ export class DatePicker extends React.Component<DatePickerProps, DatePickerState
     return this.renderCalendar();
   }
 }
+// @ts-ignore
+DatePicker.utils = {
+  getTomorrowDate,
+  get3DaysLaterDate,
```
Comment (xlreon) [inline core/components/organisms/datePicker/DatePicker.tsx]: Change variable name to exclude numerics

**Review by anuradha9712 — DISMISSED** [review]

**Review by xlreon — DISMISSED** [review]

**Review by satyamyadav — APPROVED** [review]

**Pull request merged**